src/averageAges.js

```diff
@@ -55,7 +73,23 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
```

Comment:
why not set the variable to the default value?
```suggestion
function calculateAverageAgeDiff(people, onlyWithSon = false) {
```